ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -37,9 +38,11 @@ struct MSFButtonViewButtonStyle: ButtonStyle {
                     .font(Font(tokens.textFont))
                     .fontWeight(.medium)
                     .multilineTextAlignment(.center)
+                    .frame(minHeight: tokens.textMinimumHeight)
```

Comment:
works well with larger text?

Reply:
The text size of the button doesn't change when you change the system font size. Though, we don't support the large content viewer, either, so I should probably add that.

Reply:
We have discussed this today and we've got a bug where large text is only updated after the app is restarted. I remember seeing the text size getting updated as we changed the large text config in the Settings app. We'll need to fix that bug.